eng/common/pipelines/templates/jobs/perf.yml

```diff
@@ -121,6 +121,15 @@ jobs:
       useGlobalJson: true
       workingDirectory: azure-sdk-tools
 
+  - task: UseDotNet@2
```

Comment:
This file isn't used in this repo and is part of common templates so you can undo this change.

Reply:
without this change, the repo fails to build - it needs both 10 SDKs and 9 runtimes.  What is the mechanism to ensure that the 9 runtimes can be installed?